### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SimenB**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 - [jest-extended](https://www.github.com/jest-community/jest-extended) Adds additional matchers to core API making it easy to test everything.
 - [expect-more](https://github.com/JamieMason/expect-more/tree/master/packages/expect-more-jest) A huge library of test matchers for a range of common use-cases.
+- [jest-axe](https://github.com/nickcolley/jest-axe) Custom Jest matcher for [aXe](https://axe-core.org/) for testing accessibility.
```
Comment (SimenB) [inline README.md]: /cc @nickcolley

File: README.md [hunk 2]
```diff
@@ -50,6 +51,10 @@
 
 - [jest-codemods](https://github.com/skovhus/jest-codemods) Makes it easy to migrate from other test runner and frameworks to Jest.
 
+### Library integrations
```
Comment (SimenB) [inline README.md]: I'm not sure about where else to put this, or if this title is any good...

Comment (SimenB) [reply README.md]: @searls ideas for "genre" here? 🙂

Comment (mattphillips) [reply README.md]: Mocks maybe?

**Comment by searls** [comment]
Library integrations or library plugins seems reasonable to me.

File: README.md (hunk 2 above)
Comment (searls) [reply README.md]: Library plugins or extensions seems reasonable to me. One would think that quite a few of these probably do exist (and few having to do with mocking)

File: README.md (hunk 1 above)
Comment (nickcolley) [reply README.md]: Hey! Thanks a bunch @SimenB 👍 from me

**Pull request merged**